specification/communication/data-plane/Rooms/stable/2024-04-15/communicationservicesrooms.json

```
@@ -0,0 +1,643 @@
+          "type": "string",
+          "example": "99199690362660524"
+        },
+        "createdAt": {
+          "format": "date-time",
+          "description": "The timestamp when the room was created at the server. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2021-09-06T05:55:41Z"
+        },
+        "validFrom": {
+          "format": "date-time",
+          "description": "The timestamp from when the room is open for joining. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2021-09-07T07:55:41Z"
+        },
+        "validUntil": {
+          "format": "date-time",
+          "description": "The timestamp from when the room can no longer be joined. The timestamp is in RFC3339 format: `yyyy-MM-ddTHH:mm:ssZ`.",
+          "type": "string",
+          "example": "2021-09-08T15:55:41Z"
+        },
+        "pstnDialOutEnabled": {
+          "description": "Set this flag to true if, at the time of the call, dial out to a PSTN number is enabled in a particular room. By default, this flag is set to false.",
+          "type": "boolean"
+        }
+      }
+    },
+    "RoomParticipant": {
+      "description": "A participant of the room.",
+      "required": [
+        "rawId",
+        "role"
+      ],
+      "type": "object",
+      "properties": {
+        "rawId": {
+          "description": "Raw ID representation of the communication identifier. Please refer to the following document for additional information on Raw ID. <br> https://learn.microsoft.com/azure/communication-services/concepts/identifiers?pivots=programming-language-rest#raw-id-representation",
+          "type": "string"
+        },
+        "role": {
+          "$ref": "#/definitions/Role"
+        }
+      }
+    },
+    "RoomsCollection": {
+      "description": "A collection of rooms.",
+      "required": [
+        "value"
+      ],
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "A collection of rooms",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RoomModel"
+          }
+        },
+        "nextLink": {
+          "description": "If there are more rooms that can be retrieved, the next link will be populated.",
+          "type": "string"
+        }
+      }
+    },
+    "UpdateParticipantsRequest": {
+      "description": "Participants to be updated in the room.",
+      "type": "object",
+      "properties": {
+        "participants": {
+          "description": "Participants to be updated.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/ParticipantProperties"
+          },
+          "example": {
+            "rawId1": {
+              "role": "Presenter"
+            },
+            "rawId2": {
+              "role": "Attendee"
+            },
+            "rawId3": null
```

Here too, `null` is an invalid example of "ParticipantProperties" so it should be removed.
```suggestion
            }
```

---

In PATCH operation, we use `null` to remove the entry from the collection.